### ISSUE TITLE
Improve documentation of function OktaSignIn.renderEl

### ISSUE THREAD

**Issue by mschaeffner** [okta/okta-signin-widget#424]
The documentation of the function `OktaSignIn.renderEl` should be more precise and complete regards what possible values for `type` and `status` a developer can expect in the `success` callback. See https://github.com/okta/okta-signin-widget#rendereloptions-success-error

E.g. the `type` could be `SESSION_SSO`, but this isn't documented anywhere.

Even though there is a code snippet that shows a possible implementation of the `success` callback, this code snippet is not ideal, because it contains some dead code. The lines about _OIDC response_ and _signIn.tokenManager_ are never executed, because there are `return` statements before.

**Comment by nbarbettini** [comment]
I totally agree @mschaeffner. The Github readme has become a bit gnarly. We are working on improving this right now!

**Comment by spencerkittleson** [comment]
This is still not documented after 2 years?!

**Comment by swiftone** [comment]
@spencerkittleson - Looks like this feel into the cracks.  Let me see what I can do to get this bumped up.  In the meantime, the sample code in the README does list various values you can examine: https://github.com/okta/okta-signin-widget#renderel

If you have any specific questions before we get this doc updated, feel free to post them here.

**Comment by swiftone** [comment]
Internal ref: OKTA-320763

**Comment by spencerkittleson** [comment]
@swiftone I'v read that  more than a few times.  It's the example states a property exists when it doesnt.  After the user is validated the `setCookieAndRedirect` is undefined since the session property doesnt exists.  if `next()` is done it completes the auth.  You can repo with this:
```js
signIn.renderEl(
  // Assumes there is an empty element on the page with an id of 'osw-container'
  {el: '#osw-container'},

  function success(res) {
    if (res.status === 'SUCCESS') {
      if (res.type === 'SESSION_STEP_UP') {
        // Session step up response
        // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
        // the response will contain user metadata and a stepUp object with the url of the resource
        // and a 'finish' function to navigate to that url
        console.log(res.user);
        console.log('Target resource url: ' + res.stepUp.url);
        res.stepUp.finish();
      } else {
        // If the widget is not configured for OIDC, the response will contain
        // user metadata and a sessionToken that can be converted to an Okta
        // session cookie:
        console.log(res.user);
       // BUG: session is undefined so setCookieAndRedirect fails.  
       res.session.setCookieAndRedirect('https://acme.com/app');
      }
      return;
    }

  }
);
```

**Comment by shawyu-okta** [comment]
This issue was resolved in #1473. Thank you for the report!
